content/general/pubsub.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Pub/Sub
+section: general
+index: 23
+---
+
+Pub/Sub is shorthand for the Publish/Subscribe architectural pattern, which is a popular pattern used for realtime data delivery. This messaging pattern lets any number of publishers publish data, ideally to a data channel/topic, which could be subscribed to by any number of subscribers. The important thing to note about Pub/Sub is that publishers and subscribers are completely decoupled. In addition, once subscribed, the subscribers no longer have to poll the server or data provider to check if there’s any new data that they need to be aware of; instead, they will be notified of it, as it becomes available.
```

Comment:
"instead, they will be notified of it, as it becomes available" - Redundant comma in "...notified of it, as it..."